cupy/random/__init__.py

```diff
@@ -4,10 +4,14 @@
 def bytes(length):
     """Returns random bytes.
 
+    .. note:: This function is just a wrapper for :meth:`numpy.random.bytes`.
```

Comment:
The link is broken on the CI (https://cupy--4318.org.readthedocs.build/en/4318/reference/generated/cupy.random.bytes.html).  While `numpy.random.mtrand.RandomState.bytes` is a method, `numpy.random.bytes` seems a function (`:func:`).

Reply:
`:obj:` works in the NumPy-CuPy comparison table.